clients/fides-js/src/extensions/gpp.ts

```diff
@@ -1,16 +1,56 @@
+/* eslint-disable no-underscore-dangle */
 /**
  * Extension for GPP
  *
  * Usage:
  * Include as a script tag as early as possible (even before fides.js)
  */
 
+import {
+  CmpApi,
+  CmpDisplayStatus,
+  CmpStatus,
+  SignalStatus,
+} from "@iabgpp/cmpapi";
 import { makeStub } from "../lib/gpp/stub";
+import { fidesEventToTcString } from "../lib/tcf/events";
+
+const CMP_ID = 407; // TODO: is this supposed to be the same as TCF, or is this separate?
```

Comment:
Rachel will confirm this on Monday! https://ethyca.slack.com/archives/C05E74TR0BB/p1700245618221449

Reply:
she has confirmed! We can just use our TCF CMP ID, so let's remove this comment